packages/camera/camera/CHANGELOG.md

```diff
@@ -12,6 +16,7 @@
 
 ## 0.10.4
 
+* Adds support to control video fps and bitrate.
```

Comment:
nit: small merge mistake. This line should be removed.

Reply:
Removed